src/main/java/duke/Parser/Parser.java

```diff
@@ -0,0 +1,112 @@
+package duke.Parser;
+
+/**
+ * A Parser Class does all the checking required in the programme
+ */
+
+public class Parser {
+
+
+    private final static String BYE_KEY = "bye";
+    private final static String LIST_KEY = "list";
+    private final static String DONE_KEY = "done";
+    private final static String TODO_KEY = "todo";
+    private final static String DEADLINE_KEY = "deadline";
+    private final static String EVENT_KEY = "event";
+    private final static String DELETE_KEY = "delete";
+    private final static String FIND_KEY = "find";
+
+    /**
+     * Checks for BYE Command
+     * @param checker
+     * @return boolean
+     */
```

Comment:
Please leave a line after the description.
P.S. this applies to other javadocs too. I will stop commenting on this issue:)